extensions/libraries/redcore/model/list.php

```diff
@@ -258,6 +258,9 @@ protected function populateState($ordering = null, $direction = null)
 		$app         = JFactory::getApplication();
 		$inputFilter = JFilterInput::getInstance();
 
+		// Load the parameters for frontend.
+		$params = $app->isSite() ? $app->getParams() : null;
```

Comment:
@Kixo it looks like this in fact exists since 1.5
/opt/www/public_html/kompan.us/libraries/legacy/application/application.php
	/**
	 * Is site interface?
	 *
	 * @return  boolean  True if this application is site.
	 *
	 * @since   1.5
	 * @deprecated  3.2
	 */
	public function isSite()
	{
		return $this->isClient('site');
	}